external/quarkus/playlist.xml

```diff
@@ -27,7 +27,7 @@
 		</groups>
 	</test>
 	<test>
-		<testCaseName>quarkus_java_test</testCaseName>
+		<testCaseName>quarkus_test</testCaseName>
```

Comment:
Let's leave this one alone for now, this was called quarkus_java_test because it runs in 'java mode', versus the test below it in the playlist called quarkus_native_test, which is to be run in 'native mode' (if it were enabled).